src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -38,105 +38,179 @@
 	private static final Pattern intPattern = Pattern.compile("([\\+-]?\\d+)([eE][\\+-]?\\d+)?");
 //	private static final Pattern floatPattern = Pattern.compile("([\\+-]?\\d(\\.\\d*)?|\\.\\d+)([eE][\\+-]?(\\d(\\.\\d*)?|\\.\\d+))?");
 
-	private int numberOfChannels = 2;
-	private int audioDelay = 0;
-	private int sampleFrequency = 48000;
+	public static final int BITRATE_DEFAULT = 8000;
+	public static final int NUMBEROFCHANNELS_DEFAULT = 2;
+	public static final int BITSPERSAMPLE_DEFAULT = 16;
+	public static final int AUDIODELAY_DEFAULT = 0;
+	public static final int SAMPLEFREQUENCY_DEFAULT = 48000;
+
+	private int bitRate;
+	private int numberOfChannels;
+	private int bitsperSample;
```

Comment:
I don't have problem if you want refactor it. When did you plan to do it ?
Because i'll not, as i don't feel confident to be able to do the job, and some users could wait for something to done, except if we consider DTS and PS3 as secondary...